pkg/vault/contracts/Vault.sol

```diff
@@ -1282,12 +1285,18 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
             uint256 bufferWrappedSurplus = bufferBalances.getBufferWrappedSurplus(wrappedToken);
```

Comment:
Not in the diff, but line 1282 "// surplus of underlying tokens." should be "// surplus of wrapped tokens."